From 603b8ba4eaca6e5a7137f93a42812f4ac458c830 Mon Sep 17 00:00:00 2001
From: Nizan Zorea <nzorea@marvell.com>
Date: Tue, 14 Jun 2016 15:44:02 +0300
Subject: [PATCH 504/538] dts: a70x0,a80x0: enable rtc node to dts

- cp0 node enabled (in case status property is missing the node is enabled,
  because the default status is enable).
- cp1 disabled, because there is no need of 2 rtc units.

Change-Id: I95a5768c2838c72ad517a7d2bf999ff1cd391194
Signed-off-by: Nizan Zorea <nzorea@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30464
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi | 6 ++++++
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   | 6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index dfad874..7e577b8 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -16,6 +16,12 @@
 /*
  * Generic Device Tree describing Marvell Armada CP-110 device
  */
+rtc@284000 {
+	compatible = "marvell,armada8k-rtc";
+	reg = <0x284000 0x20>, <0x284080 0x24>;
+	reg-names = "rtc", "rtc-soc";
+	status = "disabled";
+};
 
 cp110_clk_1: clk@0x440700 {
 	compatible = "marvell,armada-cp110-clock";
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index bdafa2d..26218e3 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -17,6 +17,12 @@
  * Generic Device Tree describing Marvell Armada CP-110 device
  */
 
+rtc@284000 {
+	compatible = "marvell,armada8k-rtc";
+	reg = <0x284000 0x20>, <0x284080 0x24>;
+	reg-names = "rtc", "rtc-soc";
+};
+
 cp110_clk: clk@0x440700 {
 	compatible = "marvell,armada-cp110-clock";
 	reg = <0x440700 0x4>;
-- 
1.9.1

